shared/config.ts

```diff
@@ -15,14 +15,16 @@ interface WalletConfig {
 class ConfigClass {
   api = {
     url: process.env.API_URL,
-    // TODO: use wallet address/signature?
-    address: process.env.API_ADDRESS,
-    signature: process.env.API_SIGNATURE,
   };
 
   signature = {
     api: process.env.API_SIGN_ADDRESS ?? '',
     transactionChecker: process.env.TRANSACTION_CHECKER_ADDRESS ?? '',
+    allowedMessages: {
+      regex: /^cfp-\d*-\d*:.*$/,
```

Comment:
I think this regex is not working 😉  what about this?
`/^(?:cfp|dfip)-\d+-\d+-(?:yes|no|neutral)$/`

Reply:
The sign text is very short IMO, only e.g. `cfp-2208-12-yes`

Reply:
oh I have thought, that there is the whole title in there... okay.
It was anyway just a proposed solution and not something i really really liked 😂